deploy/iroha/genesis.block

```diff
@@ -342,11 +362,23 @@
                   "roleName": "tester"
                 }
               },
+              {
+                "appendRole": {
```

Comment:
Why we need this block? We have all desired permissions in `tester`

Reply:
These question can be answered by a single phrase: Iroha permissions and domains update.
Now a one need to have all the permissions and roles to issue a new account in domain with such role etc